src/batch-client/index.ts

```diff
@@ -1573,6 +1579,33 @@ export class ZimbraBatchClient {
 				});
 			}
 		});
+	
+
+		private downloadDoc = ({ id, url ,isSecure}: any) =>
+		fetch(
+			`${this.origin}${url}`,
+			{
+				headers: {
+					...(isSecure && { 'X-Zimbra-Encoding': 'x-base64' }),
+					...(this.csrfToken && {
```

Comment:
have updated headers to this
`{
				...(this.csrfToken && {headers: {
						'X-Zimbra-Csrf-Token': this.csrfToken
					}
				}),
				credentials: 'include'
			}`

will push the code  - once reviewed ( have ran automation recently)